Symfony/CS/Config/Config.php

```diff
@@ -179,20 +179,6 @@ public function getCacheFile()
     }
 
     /**
-     * Set PHP executable.
-     *
-     * @param string|null $phpExecutable
-     *
-     * @return Config
-     */
-    public function setPhpExecutable($phpExecutable)
```

Comment:
Public methods are not "unused". They are there for people to use even if we don't.